docker/package/packages.py

```diff
@@ -63,11 +63,34 @@
         config_file="tezos-signer.conf")
 ]
 
+ledger_udev_postinst = '''
+if [ ! -f /etc/udev/rules.d/20-hw1.rules ]; then
+    cat <<EOF > /etc/udev/rules.d/20-hw1.rules
```

Comment:
One question to clean this up a bit: would it not be possible to move this into a separate file to use here and not keep all it content in `packages.py`?

Asking as it is static content and this isn't easy to look at :slightly_smiling_face:

Reply:
I agree, separate file with rules included as part of the .deb would be nice. I would go even further and suggest that there's value in making a separate .deb package for udev rules so that all tezos packages that have Ledger Nano S functionality could depend on it: client, signer, baker, endorser and accuser. Otherwise baker, for example, would only work with ledger nano s if tezos-client is also installed.